packages/tkg-clusterclass-vsphere/bundle/config/upstream/overlay.yaml

```diff
@@ -14,17 +14,6 @@ spec:
     machineInfrastructure:
       ref:
         name: tkg-vsphere-control-plane
-    machineHealthCheck:
-      #@ if data.values.ENABLE_MHC or data.values.ENABLE_MHC_CONTROL_PLANE:
-      nodeStartupTimeout: #@ data.values.NODE_STARTUP_TIMEOUT
-      unhealthyConditions:
-      - type: Ready
-        status: Unknown
-        timeout: #@ data.values.MHC_UNKNOWN_STATUS_TIMEOUT
-      - type: Ready
-        status: "False"
-        timeout: #@ data.values.MHC_FALSE_STATUS_TIMEOUT
-      #@ end
```

Comment:
This should be reverted, but also add an overlay match to add the whole machineHealthCheck. `machineHealthCheck` can't be left empty. So the whole block must be added, or none at all.